docs/getting-started/install-guide.md

```diff
@@ -6,6 +6,25 @@ This guide was last updated for v0.8.2.
 
 There are four methods for downloading and installing the Decred software. One is via dcrinstall (cross-platform), one is via the Windows Installer (Windows only of course, and the only way to get Paymetheus as of v0.8.2), another is via the precompiled binary releases (cross-platform), and the other is building the software yourself (cross-platform). The first three methods will be covered here and the fourth may be added at a later date.
 
+---
+
+## **Windows Installer for Paymetheus** 
+
+The Windows Installer (`.msi` file) is located here: [https://github.com/decred/decred-binaries/releases](https://github.com/decred/decred-binaries/releases). It will install Paymetheus to your computer's Program Files folder. Installation is pretty straightforward, but instructions are provided below:
```

Comment:
I don't do windows as you know, but is it necessary to point out that it installed to Program Files?  I thought that was the normal place for installs.